packages/app-check-interop-types/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@firebase/app-check-interop-types",
-  "private": true,
   "version": "0.1.0",
```

Comment:
change versions to `0.0.1` so minor bumps them to `0.1.0`

Reply:
Done.